web/styles/left_sidebar.css

```diff
@@ -1353,7 +1357,14 @@ li.topic-list-item {
     overflow: hidden;
 }
 
-.topic-list-filter {
+.direct-messages-search-section {
+    padding-left: 10px;
```

Comment:
I'm not in love with the idea of adding more magical padding values here. Likewise maintaining the calculated width below, which ought to be replaced by a judicious use of flexbox or possibly CSS grid.

Reply:
One possible solution might be to have the DM section as one CSS grid container. And that might reduce the need for the calculated widths and padding. 

Do we want to have a intermediate solution here or address this in a different issue?